### PR TITLE
Code for v2lambda

### PR DESCRIPTION
**Pull request by prottayCMT**

### CODE REVIEW

**Comment by prottayCMT** [comment]
@BongHwi your approval...

**Review by BongHwi — APPROVED** [review]

File: PWGLF/Tasks/Resonances/lambdav2.cxx [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+      fMultCutHigh = new TF1("fMultCutHigh", "[0]+[1]*x+[2]*x*x+[3]*x*x*x + 3.*([4]+[5]*x)", 0, 100);
+      fMultCutHigh->SetParameters(1893.94, -53.86, 0.502913, -0.0015122, 109.625, -1.19253);
+      fMultMultPVCut = new TF1("fMultMultPVCut", "[0]+[1]*x+[2]*x*x", 0, 5000);
+      fMultMultPVCut->SetParameters(-0.1, 0.785, -4.7e-05);
+    }
+  }
+
+  double massLambda = TDatabasePDG::Instance()
+                        ->GetParticle(kLambda0)
+                        ->Mass(); // FIXME: Get from the common header
```
Comment (vkucera) [inline PWGLF/Tasks/Resonances/lambdav2.cxx:211]: Please don't duplicate code that should be fixed.

File: PWGLF/Tasks/Resonances/lambdav2.cxx [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+  {
+    double result = phi;
+    while (result < 0) {
+      result = result + 2. * TMath::Pi() / 2;
+    }
+    while (result > 2. * TMath::Pi() / 2) {
+      result = result - 2. * TMath::Pi() / 2;
+    }
+    return result;
+  }
```
Comment (vkucera) [inline PWGLF/Tasks/Resonances/lambdav2.cxx:369]: There is `RecoDecay::constrainAngle` for this.

File: PWGLF/Tasks/Resonances/lambdav2.cxx [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+
+    const float pT = candidate.pt();
+    const float tranRad = candidate.v0radius();
+    const float dcaDaughv0 = candidate.dcaV0daughters();
+    const float cpav0 = candidate.v0cosPA();
+    float CtauK0s = candidate.distovertotmom(collision.posX(), collision.posY(),
+                                             collision.posZ()) *
+                    TDatabasePDG::Instance()
+                      ->GetParticle(kLambda0)
+                      ->Mass(); // FIXME: Get from the common header
```
Comment (vkucera) [inline PWGLF/Tasks/Resonances/lambdav2.cxx:282]: Again

**Pull request merged**